server-phoenix/mix.exs

```diff
@@ -50,7 +50,8 @@ defmodule Helios.MixProject do
       {:jason, "~> 1.2"},
       {:plug_cowboy, "~> 2.5"},
       {:absinthe, "~> 1.7"},
-      {:absinthe_plug, "~> 1.5"}
+      {:absinthe_plug, "~> 1.5"},
+      {:dotenv, "~> 3.0.0", only: [:dev, :test]}
```

Comment:
I'm unclear how this works in production if we have something like `Dotenv.get("<ENV_VARIABLE>")` in code.

Reply:
I can add the library for all environments by leaving out the "only" option. Are these env variables loaded into heroku directly for production? (through dashboard settings or the cli)

Reply:
They are. My question is how will `Dotenv.get` function in the code in production if its only available for `:dev` and `:test`?

Reply:
I can remove the "only" option like this:
`{:dotenv, "~> 3.0.0"}`
so then it'll be available in all environments.